ui/components/app/edit-gas-fee-popover/edit-gas-tooltip.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { PRIORITY_LEVELS } from '../../../../shared/constants/gas';
+import { COLORS, FONT_WEIGHT } from '../../../helpers/constants/design-system';
+import Typography from '../../ui/typography';
+import InfoTooltip from '../../ui/info-tooltip';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+const EditGasToolTip = ({priorityLevel, gasContext}) => {
+  const {
+    gasLimit,
+    maxFeePerGas,
+    maxPriorityFeePerGas,
+    maxFeePerGasValue,
+    maxPriorityFeePerGasValue,
+    origin,
+  } = gasContext;
+
+  const t = useI18nContext();
+
+  const toolTipMessage = () => {
+    switch (priorityLevel) {
+      case PRIORITY_LEVELS.LOW:
+        return t('lowGasSettingToolTipMessage', [
+          <span key={priorityLevel}>
+            <b>{t('low')}</b>
+          </span>,
+        ]);
+      case PRIORITY_LEVELS.MEDIUM:
+        return t('mediumGasSettingToolTipMessage', [
+          <span key={priorityLevel}>
+            <b>{t('medium')}</b>
+          </span>,
+        ]);
+      case PRIORITY_LEVELS.HIGH:
+        return t('highGasSettingToolTipMessage', [
+          <span key={priorityLevel}>
+            <b>{t('high')}</b>
+          </span>,
+        ]);
+      case PRIORITY_LEVELS.CUSTOM:
+        return t('customGasSettingToolTipMessage', [
+          <span key={priorityLevel}>
+            <b>{t('custom')}</b>
+          </span>,
+        ]);
```

Comment:
First 4 cases can be reduced to 

```
case PRIORITY_LEVELS.CUSTOM:
        return t(`${priorityLevel}GasSettingToolTipMessage`, [
          <span key={priorityLevel}>
            <b>{t(priorityLevel)}</b>
          </span>,
        ]);
```

Reply:
I originally had it like this but `yarn verify-locales` didn't recognize that these keys were being used... so had to use this much less satisfying and concise approach 😢